.bin/cleanup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set +ex
```

Comment:
clarifying, you wish to _disable_ errexit and xtrace, or did you intend set -ex?

I don't usually set xtrace, but don't see harm in more debugging.

Reply:
Probably a typo for -ex